scripts/restructure

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/perl
+
+package PerlWeeklyChallenge;
```

Comment:
I would suggest build a library that does collective things and can be tested separately. In the new directory structure, create folder "lib/PWC/Utils"
Under this folder create package DataMigration.pm and DataMigration/Option.pm

For reference, please checkout one of my CPAN module:

https://metacpan.org/pod/Map::Tube::CLI

Also get into the habit of creating unit test for everything. For this I would suggest create new folder as below

"t/datamigration/00-run.t"